docker/dockerhub_to_novi_ecr.sh

```diff
@@ -60,25 +60,8 @@ docker pull --disable-content-trust=false docker.io/libra/client:"$DOCKERHUB_TAG
 
 export DOCKER_CONTENT_TRUST=0
 
-#Push to existing locations
-docker tag libra/init:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra_init:"$OUTPUT_TAG"
-docker tag libra/faucet:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra_faucet:"$OUTPUT_TAG"
-docker tag libra/tools:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra_tools:"$OUTPUT_TAG"
-docker tag libra/validator:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra_validator:"$OUTPUT_TAG"
-docker tag libra/validator_tcb:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra_safety_rules:"$OUTPUT_TAG"
-docker tag libra/cluster_test:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra_cluster_test:"$OUTPUT_TAG"
-docker tag libra/client:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra_client:"$OUTPUT_TAG"
 
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_init:"$OUTPUT_TAG"
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_faucet:"$OUTPUT_TAG"
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_tools:"$OUTPUT_TAG"
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_validator:"$OUTPUT_TAG"
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_safety_rules:"$OUTPUT_TAG"
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_cluster_test:"$OUTPUT_TAG"
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_client:"$OUTPUT_TAG"
-
-
-#Push the proper locations to novi ecr.
+# Push to old Novi ECR
 docker tag libra/init:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra/init:"$OUTPUT_TAG"
 docker tag libra/faucet:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra/faucet:"$OUTPUT_TAG"
 docker tag libra/tools:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra/tools:"$OUTPUT_TAG"
```

Comment:
yikes, I just landed https://github.com/libra/diem/pull/6846 this afternoon which removed the libra/* images. I guessed we still need them downstream.

Reply:
I haven't switched the ECR repos yet, so yes